manifest.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: powerdummy-populator
+  namespace: relationships
+  labels:
+    app: powerdummy
+    team: relationships
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
```

Comment:
really? Unless you really need this behavior (never more than 1 pod, zero during updates), you should leave this section with the defautl strategy (just remove it)

Reply:
fixed